monitoring/monitoring_client.py

```diff
@@ -0,0 +1,115 @@
+"""
```

Comment:
i think this is supposed to be a rename. note that there are changes to this file.